core/build.cmd

```diff
@@ -0,0 +1,2 @@
+@ECHO OFF
+PowerShell -NoProfile -NoLogo -ExecutionPolicy unrestricted -Command "[System.Threading.Thread]::CurrentThread.CurrentCulture = ''; [System.Threading.Thread]::CurrentThread.CurrentUICulture = '';& '%~dp0build.ps1' %*"
```

Comment:
This will work fine for experiments and first previews - to make this all working. But in the long run this needs to be integrated into our full build system. We do (and need to do) much more than just build and package binaries into a NuGet nupkg.